tests/queries/0_stateless/01934_constexpr_aggregate_function_parameters.sql

```diff
@@ -5,7 +5,7 @@ SELECT groupArray()(number) FROM numbers(10); -- { serverError 36 }
 SELECT groupArray(NULL)(number) FROM numbers(10); -- { serverError 36 }
 SELECT groupArray(NULL + NULL)(number) FROM numbers(10); -- { serverError 36 }
 SELECT groupArray([])(number) FROM numbers(10); -- { serverError 36 }
-SELECT groupArray(throwIf(1))(number) FROM numbers(10); -- { serverError 395 }
+SELECT groupArray(throwIf(1))(number) FROM numbers(10); -- { serverError 134 }
```

Comment:
Now it returns another error code because `throwIf()` also forbids const folding and cannot be evaluated as a constant now.